queries/weather.py

```diff
@@ -102,6 +104,12 @@
 QWeatherCapitalRegion →
     "á" "höfuðborgarsvæðinu" | "í" "reykjavík"
 
+QWeatherAnyLoc →
+    QWeatherCountry | QWeatherCapitalRegion | QWeatherOpenLoc
```

Comment:
These alternatives look like they might be ambiguous with respect to each other. Might be good to have `>` signs between alternatives in the production, rather than `|`, to indicate absolute priority. Or, otherwise, to give them different scores with the `$score()` pragma.